CHANGELOG.md

```diff
@@ -1,6 +1,50 @@
 # Changelog
 
-## [v1.0.29](https://github.com/stargate/stargate/tree/v1.0.29) (2021-06-29)
+## [v1.0.30](https://github.com/stargate/stargate/tree/v1.0.30) (2021-07-19)
```

Comment:
I'm not sure this is correct.... did we lose the section about 1.0.29?

Reply:
Looks like we just lost the header

Reply:
Fixed. I just ran the script.